x509/x509.go

```diff
@@ -1862,15 +1866,25 @@ func ParseCertificates(asn1Data []byte) ([]*Certificate, error) {
 		v = append(v, cert)
 	}
 
+	var nfe NonFatalErrors
 	ret := make([]*Certificate, len(v))
 	for i, ci := range v {
 		cert, err := parseCertificate(ci)
 		if err != nil {
-			return nil, err
+			if errs, ok := err.(NonFatalErrors); !ok {
+				return nil, err
+			} else {
+				for _, err := range errs.Errors {
+					nfe.Errors = append(nfe.Errors, err)
```

Comment:
Can't you do this all in one with append(nfe.Errors, errs.Errors...)?

Reply:
True - fixed.